hub/server.go

```diff
@@ -29,6 +29,18 @@ func (h *Hub) Serve() {
 		WriteTimeout: h.config.GetDuration("write_timeout"),
 	}
 
+	if h.config.GetBool("metrics_enabled") {
+		addr := h.config.GetString("metrics_addr")
+
+		h.metricsServer = &http.Server{
+			Addr:    addr,
+			Handler: h.metricsHandler(),
+		}
+
+		log.WithFields(log.Fields{"addr": addr}).Info("Mercure metrics started")
+		go h.metricsServer.ListenAndServe()
```

Comment:
This server must be shutdown too in `listenShutdown()` is started.

Reply:
Done